File: isem-finanzas/common/backend/database/Models/Modelos/RequerimientoModel.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations.Schema;
using System.Text;

namespace Models.Modelos
{
    public class RequerimientoModel
    {
        public int? id_requerimiento { get; set; }
        public int? id_oficio { get; set; }
        public int? id_auditoria { get; set; }
        //[NotMapped] public int? id_asignado { get; set; }
        public string descripcion { get; set; }
        public string fecha_recepcion { get; set; }
        //public string fecha_asignacion { get; set; }
        public bool? activo { get; set; }
        public string num_oficio { get; set; }
    // public string nombre_asignado { get; set; }
        public int? id_estatus { get; set; }
        public string nombre_estatus { get; set; }
        public int? id_usuario_requerimiento { get; set; }
        public string nombre_usuario_requerimiento { get; set; }
        public string lista_asignados { get; set; }
        public int? id_requerimiento_asignado { get; set; }
        public bool? aceptado { get; set; }
        [NotMapped] public string observaciones { get; set; }
        [NotMapped] public int tipoOperacion { get; set; }
    }

}

File: isem-finanzas-web/src/app/@finanzas/@auditorias/models/requerimiento.model.ts
export class RequerimientoModel {
  id_requerimiento?: number;
  id_oficio?: number;
  id_auditoria?: number;
  //id_asignado?: number;
  descripcion?: string;
  fecha_recepcion?: string;
  //fecha_asignacion?: string;
  activo?: boolean;
  num_oficio?: string;
  //nombre_asignado?: string;
  id_estatus?: number;
  nombre_estatus?: string;
  id_usuario_requerimiento?: number;
  nombre_usuario_requerimiento?: string;
  lista_asignados?: string;
  observaciones?: string;
  accion?: string;
  asignados?: string;
  id_requerimiento_asignado?: number;
  aceptado?: boolean;
}


